src/components/header/Header.tsx

```diff
@@ -0,0 +1,16 @@
+import './Header.css'
+import { BiSearchAlt, BiFilterAlt } from 'react-icons/bi'
+import { AiOutlineSortDescending } from 'react-icons/ai'
```

Comment:
You could use alias to have more readable names for these components eg (decide yourself whether you want to use it):
```suggestion
import { BiSearchAlt as SearchIcon, BiFilterAlt as FilterIcon } from 'react-icons/bi'
import { AiOutlineSortDescending as SortIcon } from 'react-icons/ai'
```